index.html

```diff
@@ -3,37 +3,81 @@
 <head>
     <meta charset="utf-8" />
     <title>Google calendar</title>
+    <link rel="shortcut icon" href="C:\Users\rama\workspace\calendar\favicon.ico" /> <!--added this so that we don't get a server side error for the favicon.ico image-->
```

Comment:
please remove this, this should be handled by the server side logic, add a new route and return an empty body